source/documentation/reference/cloud-platform-support.md

```diff
@@ -0,0 +1,71 @@
+# Cloud Platform Support
+
+## Background
+
+The Cloud Platform Team has built a new platform to replace Template Deploy. We are employing modern tools and techniques to enable teams to develop and support their modern cloud native software and infrastructure.
+
+This also provides us with an opportunity to enable teams, through self-service, to develop and support products the way they want to, without having to rely on the Cloud Platform Team for assistance.
+
+This new flexibility changes the way in which the Cloud Platform team provides support to our users.
+
+## Cloud Platform Team Support Processes
+
+The Cloud Platform [Operational Processes](#operational-processes.md) details how we will offer support to our users for the new platform.
+
+This includes support hours, out-of-hours support, operational communication with users, incident management, and how we measure performance. 
+
+## What does Cloud Platform Team support
+
+Below is a diagram of the Cloud Platform.
+
+![Cloud Platform Diagram](../images/cloud-platform-diagram.png)
```

Comment:
This diagram is present in the master branch, but not in this branch. I think you need to `git rebase master`

Reply:
image now showing